File: dgt/modelo/src/main/java/com/ipartek/appMultas/modelo/pojo/Agente.java
package com.ipartek.appMultas.modelo.pojo;

public class Agente {
	private Long id;
	private String nombre;
	private String placa;
	

	public Agente() {
		super();
	}

	public Agente(Long id, String nombre, String placa) {
		this();
	
		setId(id);
		setNombre(nombre);
		setPlaca(placa);
	
		
	}

	public Long getId() {
		return id;
	}

	public void setId(Long id) {
		this.id = id;
	}

	public String getNombre() {
		return nombre;
	}

	public void setNombre(String nombre) {
		this.nombre = nombre;
	}

	public String getPlaca() {
		return placa;
	}

	public void setPlaca(String placa) {
		this.placa = placa;
	}

	

	
	

	@Override
	public String toString() {
		return "Agente [id=" + id + ", nombre=" + nombre + ", placa=" + placa + ", id_departamento=]";
	}
}

File: dgt/angular/src/app/model/agente.ts
export class Agente {

    private _id: number;
    private _nombre: string;
    private _placa: string;
    private _id_departamento: number;
    private _password: string;

    constructor(
        id: number = -1,
        nombre: string,
        placa: string,
        id_departamento: number,
        password: string
    ) {
        this._id = id;
        this._nombre = nombre;
        this._placa = placa;
        this._id_departamento = id_departamento;
        this._password = password;
    }

    public get id(): number {
        return this._id;
    }
    public set id(value: number) {
        this._id = value;
    }
    public get nombre(): string {
        return this._nombre;
    }
    public set nombre(value: string) {
        this._nombre = value;
    }
    public get placa(): string {
        return this._placa;
    }
    public set placa(value: string) {
        this._placa = value;
    }
    public get id_departamento(): number {
        return this._id_departamento;
    }
    public set id_departamento(value: number) {
        this._id_departamento = value;
    }
    public get password(): string {
        return this._password;
    }
    public set password(value: string) {
        this._password = value;
    }
}
